napari/_app_model/_app.py

```diff
@@ -1,20 +1,26 @@
 from __future__ import annotations
 
 from functools import lru_cache
-from warnings import warn
 
 from app_model import Application
 
 from napari._app_model.actions._layerlist_context_actions import (
     LAYERLIST_CONTEXT_ACTIONS,
     LAYERLIST_CONTEXT_SUBMENUS,
 )
-from napari.utils.translations import trans
 
 APP_NAME = 'napari'
 
 
 class NapariApplication(Application):
+    """A singleton class representing the Napari application.
```

Comment:
```suggestion
    """A singleton (per name) class representing the Napari application.
```

To suggest that it is not pure singleton, but keep one per application name.